libvips/iofuncs/init.c

```diff
@@ -648,6 +648,13 @@ vips_init( const char *argv0 )
 	if( g_getenv( "VIPS_BLOCK_UNTRUSTED" ) )
 		vips_block_untrusted_set( TRUE );
 
+	/* Since it's so easy to forget the "--buildtype release" arg to meson.
+	 */
+#ifdef DEBUG_BUILD
```

Comment:
Given that we use Meson's `b_ndebug=if-release`.
https://github.com/libvips/libvips/blob/f807d7b1aaa09d10e4b38583fb417fb7f74b5e3c/meson.build#L9-L10

Perhaps it would be simpler to just check for `NDEBUG` instead?:
```suggestion
#ifndef NDEBUG
```

Reply:
We could do, though of course someone could build with `-DNDEBUG=1`, so I thought maybe a define that is *just* used for signalling debug mode might be safer.